addons/osfstorage/migrations/0001_initial.py

```diff
@@ -1,28 +1,74 @@
-# -*- coding: utf-8 -*-
-# Generated by Django 1.9 on 2017-03-23 20:34
-from __future__ import unicode_literals
+# Generated by Django 3.2 on 2022-08-08 15:38
 
+import addons.base.models
+from django.conf import settings
 from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
 import osf.models.base
+import osf.utils.datetime_aware_jsonfield
+import osf.utils.fields
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
     dependencies = [
+        ('osf', '0001_initial'),
```

Comment:
Region models in the osfstorage addon app depend on the `osf` app migrations. I squashed these migrations just to simplify the behavior.